src/app/pages/storage/modules/disks/components/manual-test-dialog/manual-test-dialog.component.html

```diff
@@ -50,6 +50,25 @@ <h1 matDialogTitle>{{ 'Manual S.M.A.R.T. Test' | translate }}</h1>
   </ix-form-actions>
 </form>
 <ng-template #showResults>
+  <div *ngIf="!endedTests" class="progress-bar-panel">
+    <div
+      fxLayout="row"
+      fxLayoutAlign="space-between baseline"
+      fxLayoutGap="8px"
+    >
+      <span>{{ subName }}</span>
+      <small>
+        {{ progressTotalPercent || 0 | number: '1.2-2' }}%
```

Comment:
Wondering if the `percent` pipe is applicable here
https://angular.io/api/common/PercentPipe